src/addRemove.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
+import { listArray } from './index.js';
+
+export class ListArrayClass {
```

Comment:
- [ ] great job creating the add and delete function 👍 however in this milestone the requirements also mention implementing a function for editing task descriptions. please do so before requesting another code review.